learning-objectives/intl/es.yml

```diff
@@ -174,6 +174,9 @@ js/data-types/objects:
 
 js/data-types/primitive-vs-non-primitive:
   title: Diferenciar entre tipos de datos primitivos y no primitivos
+  links:
+    - title: "datos atómicos vs datos estructurados"
+      url: https://www.todojs.com/tipos-datos-javascript-es6/
```

Comment:
Este link en particular ha sido revisado por ustedes @ssinuco @GenesisMauries ?

Como para saber que está correcta la info que tiene, yo lo miré a la rápida, me encontré con cosas como `var` que intentamos esquivar normalmente, pero les parece que de igual forma es la mejor lectura para ello? :blush:

Reply:
@ssinuco Sugiero utilizar este https://curriculum.laboratoria.la/es/topics/javascript/basics/values-variables-and-types